cypress/support/commands.js

```diff
@@ -41,6 +41,7 @@ Cypress.Commands.add("setupGroup", (students, group) => {
     let i=0, j=0;
 
     for (i=0;i<students.length;i++) {
+        cy.wait(2000)
```

Comment:
@kswenson I remember we talked about this `wait`, do you remember where we landed on that?